zone_bounds:
 - zone_name: zone_1
   x_min: 0.5
   x_max: 1.5
   y_min: -1.2
   y_max: 0.0
   next_location_gen_mode: 1
 - zone_name: zone_2
   x_min: 0.5
   x_max: 1.5
   y_min: 0.0
   y_max: 1.2
   next_location_gen_mode: 0
place_zone:
  frame_id: base_link
  pose_candidates: 8
  orientation_axis:
    x: 0
    y: 0
    z: 1
  release_distance: 0.02
  next_location:
    min_spacing: 0.05
    max_spacing: 0.08
